src/App.js

```diff
@@ -5,13 +5,18 @@ import AddTodoForm from "./AddTodoForm";
 
 
 function App() {
-  return (
-      <div>
-          <h1> Todo List </h1>
-          <AddTodoForm></AddTodoForm>
-          <TodoList></TodoList>
-      </div>
-  );
+
+    const [newTodo, setNewTodo] = React.useState('');
```

Comment:
nit: you don't need to specify an argument to `useState` the argument is the default value, so if you leave it blank that will work just fine. Try it out! Also try and see what happens if you enter in some text as the argument